.gitignore

```diff
@@ -28,6 +28,8 @@ components/**/ios/Generated
 
 # XCFramework artifact
 megazords/ios-rust/MozillaRustComponents.xcframework*
+megazords/ios-rust/focus/FocusRustComponents.xcframework*
```

Comment:
hmmm, I wonder if it'll be worth it eventually to move the firefox into `fx-ios` or something, but more of a musing and definitely not in scope for this PR